helm/freenas-provisioner/templates/storageclass.yaml

```diff
@@ -0,0 +1,22 @@
+---
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: freenas-nfs
+  labels:
+{{ include "freenas-provisioner.labels" . | nindent 4 }}
```

Comment:
Just to be perfect, indentation should be 4 spaces here and starting with a `-`char like so:
```
     {{- include "freenas-provisioner.labels" . | nindent 4 }} 
```
But I am being very perfectionist here ;)

Reply:
This is how `helm create` builds it so I'm a little hesitant to change this.

Reply:
I guess, this is kind of a bug in `helm create` command side. This is not a big deal anyway.